test/end2end-test/test_mobilenetv2-1.0_onnx.js

```diff
@@ -0,0 +1,31 @@
+describe('End2End Test/ONNX Models', function() {
+  const assert = chai.assert;
+  const modelName = 'mobilenetv2-1.0';
+  const modelFile = `./end2end-test/resources/${modelName}/${modelName}.onnx`;
+  let rawModel;
+
+  before(async function () {
+    this.timeout(120000);
```

Comment:
this.timeout(120000);

What is this timeout for?

Reply:
The function **loadOnnxModel**  in next #L9 as following code is an async function, 
```javascript
    rawModel = await loadOnnxModel(modelFile);
```
this #L8 timeout code makes model file load simultaneously, or we would fail to get concrete **rawModel** for var **kwargs** which is a parameter of **OnnxModelImporter** constructor.
```javascript
      let kwargs = {
        rawModel: rawModel,
        backend: options.backend,
        prefer: options.prefer,
        softmax: false,
      };
      let model = new OnnxModelImporter(kwargs);
```